Docker/README.md

```diff
@@ -0,0 +1,41 @@
+# Docker files details
+
+## Development environment:
+
+### Pre-requisite:
+
+The "EdFi.Ods.AdminApi" application folder and "Nuget.config" file will be
+copied over, either manually or through the execution of a script(build.ps1
+-Command "CopyApplicationFilesToDockerContext"), to the "Application" folder
+within the "Docker" directory. 
```

Comment:
No need for a colon after a header (line 3, 5, and 25).

Would it make sense to move the "Pre-requisite" block below the two items? Currently, it looks like the two items below are part of the pre-requisite.

Please add code ticks around the PowerShell statement: `build.ps1 -Command "CopyApplicationFilesToDockerContext"`